spec/features/account_history_spec.rb

```diff
@@ -28,6 +31,8 @@
 
   scenario 'viewing account history' do
     expect(page).to have_content(t('event_types.account_created'))
+    expect(page).to have_content(t('event_types.usps_mail_sent'))
+    expect(user_event_types).to match_array(%w[account_created usps_mail_sent usps_mail_sent])
```

Comment:
Since this is testing what shows up on the user's account page, if the user has requested multiple USPS letters, what do you think about testing that both of those events show up with different timestamps? Testing that `user.events.pluck(:event_type)` contains multiple `usps_mail_sent` events is not enough because that only tests what's in the DB. It doesn't test that the view is iterating through all events.

Reply:
using a hypothetical event timeline, testing for presence of decorated event timestamps.